extension/config/tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
NIT: remove comment?

Reply:
that's actually a jsdoc that was automatically added .. i would rather keep it